deploy/Chart/templates/controller/webhook.yaml

```diff
@@ -16,13 +16,9 @@ metadata:
     app.kubernetes.io/name: controller
     app.kubernetes.io/part-of: radius
 data:
-  {{ if $existingSecret }}tls.crt: {{ index $existingSecret.data "tls.crt" }}
-  {{ else }}tls.crt: {{ b64enc $cert.Cert }}
-  {{ end }}
-
-  {{ if $existingSecret }}tls.key: {{ index $existingSecret.data "tls.key" }}
-  {{ else }}tls.key: {{ b64enc $cert.Key }}
-  {{ end }}
+  tls.crt: {{ include "secrets.lookup" (dict "secret" "controller-cert" "namespace" .Release.Namespace "key" "tls.crt" "defaultValue" $cert.Cert) }}
+  tls.key: {{ include "secrets.lookup" (dict "secret" "controller-cert" "namespace" .Release.Namespace "key" "tls.key" "defaultValue" $cert.Key) }}
+  ca.crt: {{ include "secrets.lookup" (dict "secret" "controller-cert" "namespace" .Release.Namespace "key" "ca.crt" "defaultValue" $ca.Cert) }}
```

Comment:
Why do we need this ? this is dupe with tls.crt

Reply:
`tls.crt` uses `$cert.Cert` while `ca.crt` uses `$ca.Cert`. They are different.

Reply:
`ca.crt` is also added here: https://github.com/bitnami/charts/blob/882140cfc5318420fdf563b77201f7ee5f67ca8f/bitnami/rabbitmq-cluster-operator/templates/messaging-topology-operator/validating-webhook-configuration.yaml#L29. That is what we use for the **caBundle**.

We need a way to access existing `ca.crt` to use it in the **caBundle**.